packages/embed-grid/index.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <base href="./" />
```

Comment:
This should still be `BASE_URL`.

Reply:
I was just trying to make this simple (using Vite default config). I don't see any reason `BASE_URL` needs to be easily changeable at build time as we don't support anything other than `./`

Reply:
Yea fair